spec/aggregate_root_spec.rb

```diff
@@ -3,16 +3,18 @@
 class Order
   include RailsEventStore::AggregateRoot
 
+  attr_accessor :id, :status
```

Comment:
This should be also private. No state outside aggregate :) And this change should go in separate PR. Would you like to move it out from this PR ? I think new PR with this change could be a good starting point for a discussion how to do it better.